components/form.js

```diff
@@ -139,6 +140,94 @@ function setNativeValue (textarea, value) {
   textarea.dispatchEvent(new Event('input', { bubbles: true, value }))
 }
 
+function useEntityAutocomplete ({
```

Comment:
this is a hook, but it was named like a component `EntityAutocomplete` which made it confusing